### ISSUE TITLE
Publish docker image to ECR Public

### ISSUE THREAD

**Issue by rothgar** [VictoriaMetrics/VictoriaMetrics#4116]
### Describe the solution you'd like

I would like the official victoria metrics container images to also be published on gallery.ecr.aws

### Describe alternatives you've considered

I have used dockerhub but their quotas are lower and it takes longer to pull images in some AWS regions.

**Comment by denisgolius** [comment]
Hi @rothgar !

1. What do we need from our side to register on it?
2. Where to find a price list, if any?
3. Does `gallery.ecr.aws` have statistics for downloads?

**Comment by tenmozes** [comment]
hi, @rothgar is the main problem the quota usage from the Docker hub. If so we evaluated adding VictoriaMetrics to the quay to prevent annoying limitations from the Docker side? do you see any benefits to use ECR vs quay?

**Comment by rothgar** [comment]
I haven't personally run into quota limits (I just found victoriametrics last week) but I was thinking about running it in Kubernetes clusters and I work at AWS. I always prefer to host containers I use in ECR private or public to avoid adding costs for the upstream uprojects and decrease external dependencies.

Personally, I don't every intentionally pull images from quay or go there to look for images.

**Comment by joryirving** [comment]
Are there any updates on this? I keep running into dockerhub quotas, specifically due to Victoria Metrics. I'm currently investigating either a pull-through cache, a reflector like spegel, or just pushing the image to our own ECR repo, but it'd be wonderful to see this on quay, or even ghcr.

**Comment by neilkuan** [comment]
Hi @denisgolius .
Are there any updates on this?
https://docs.docker.com/docker-hub/usage/

Starting April 1, 2025, all users with a Pro, Team, or Business subscription will have unlimited Docker Hub pulls with fair use. Unauthenticated users and users with a free Personal account have the following pull limits:

Unauthenticated users: 10 pulls/hour
Authenticated users with a free account: 100 pulls/hour

**Comment by hagen1778** [comment]
Yes, we are considering pushing our images to alternative registries. So far we're evaluating https://quay.io/, https://ghcr.io, https://aws.amazon.com/ecr/.

**Comment by zekker6** [comment]
A quick update on this: 
We've decided to give quay a go and currently working on mirroring existing images to repo here - https://quay.io/organization/victoriametrics
Future releases will be published to both docker.io and quay.io.

**Comment by hagen1778** [comment]
VictoriaMetrics and VictoriaLogs images are now published to alternative registry: https://quay.io/organization/victoriametrics 
We will publish new releases and updates to both: [docker hub](https://hub.docker.com/u/victoriametrics) and [quay.io](https://quay.io/organization/victoriametrics). Thanks to @zekker6 for implementing this!

Closing this ticket as done.

**Comment by alexhampu** [comment]
It looks like for victoria-logs the latest update on quay.io was on February 28th: https://quay.io/repository/victoriametrics/victoria-logs?tab=tags.

Can we get an automated update on this image as well? Thank you.

**Comment by zekker6** [comment]
Hello @alexhampu, thank you for noticing!
I've synced images between docker and quay registries and we will fix release process to make sure images will be published to both as usual.
